test/fixtures/implicit-returns/loop.js

```diff
@@ -1,11 +1,11 @@
 fn(x) ->
-  for i from 0 til 10:
+  for let i=0; i<10; i++:
```

Comment:
Can do, but will defer changes not related to actually-broken tests until after functionality is complete.